packages/daemon/src/service/pipeline.ts

```diff
@@ -77,15 +67,15 @@ export class PipelineService {
   pluginManager: PluginManager;
 
   createPipeline(config: PipelineDB): Promise<PipelineModel> {
-    if (!config.id || validate(config.id)) {
-      config.id = uuidv1();
+    if (typeof config.id !== 'string') {
+      config.id = generate();
     }
     return this.pipeline.create(config);
   }
 
-  async getPipeline(id: string): Promise<PipelineModel> {
+  async getPipeline(idOrName: string): Promise<PipelineModel> {
     return this.pipeline.findOne({
-      where: getIdOrName(id)
+      where: { [Op.or]: [ { id: idOrName }, {name: idOrName } ] }
```

Comment:
```suggestion
      where: { [Op.or]: [ { id: idOrName }, { name: idOrName } ] }
```

Or in multiple lines.